libs/payments/ui/src/lib/server/components/PurchaseDetails/index.tsx

```diff
@@ -189,7 +199,37 @@ export async function PurchaseDetails(props: PurchaseDetailsProps) {
           </span>
         </li>
       </ul>
-      {/* TODO - Add InfoBox as part of Coupon Form - Consider adding as child component */}
+
+      {!discountType || discountType === 'forever' ? null : discountEnd ? (
+        <div className="flex items-center justify-center gap-2 text-green-900 pt-2 pb-6 font-medium">
+          <Image src={infoLogo} alt="" />
+          <div>
+            {l10n.getString(
+              'next-coupon-success-with-date',
```

Comment:
I tried it out in another language, and this string didn't translate as expected. Is this the correct l10n id?